lib/solr.js

```diff
@@ -82,7 +82,8 @@ function Client(options){
       path : options.path || '/solr',
       agent : options.agent,
       secure : options.secure || false,
-      bigint : options.bigint || false
+      bigint : options.bigint || false,
+      get_max_request_entity_size: options.get_max_request_entity_size || 2048
```

Comment:
Default it to false, so that this is not a breaking change 
get_max_request_entity_size: options.get_max_request_entity_size || false